src/FlipMove.js

```diff
@@ -450,32 +475,27 @@ class FlipMove extends Component {
 
       // If the child element returns null, we need to avoid trying to
       // account for it
-      if (!childData.domNode) {
+      if (!childData.domNode || !child) {
         return;
       }
 
       childData.boundingBox = getRelativeBoundingBox({
-        childData,
-        parentData: this.parentData,
+        childDomNode: childData.domNode,
+        parentDomNode,
         getPosition: this.props.getPosition,
       });
     });
   }
 
-  computeInitialStyles(child) {
-    const noAnimationRequestedForThisEvent = (
-      (child.appearing && !this.props.appearAnimation) ||
-      (child.entering && !this.props.enterAnimation) ||
-      (child.leaving && !this.props.leaveAnimation)
-    );
-
-    if (noAnimationRequestedForThisEvent) {
-      return {};
-    }
```

Comment:
I didn' kill it, just splitted and moved below. Flow isn't clever enough to find out that if `noAnimation...` is falsy, and `child.appearing` is truthy,  `this.props.appearAnimation` cannot be null